modules/common.nix

```diff
@@ -0,0 +1,78 @@
+moduleConfig:
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  svcName = "vscode-server-fixup";
+  cfg = config.services.${svcName};
+in
+{
+  options.services.${svcName} = {
+    enable = mkEnableOption "automatic fixups for the VS Code remote SSH service";
+
+    nodejsPackage = mkOption {
+      type = types.package;
+      description = ''  
+        Which Node.js derivation to use.
+      '';
+      default = pkgs.nodejs-14_x;
+      defaultText = literalExpression "pkgs.nodejs-14_x";
+    };
+
+    ripgrepPackage = mkOption {
+      type = types.package;
+      description = ''  
+        Which ripgrep derivation to use.
+      '';
+      default = pkgs.ripgrep;
+      defaultText = literalExpression "pkgs.ripgrep";
+    };
+
+    watchDirs = mkOption {
+      type = types.listOf types.str;
+      description = ''
+        List of absolute directory paths to watch to apply automatic fixes.
+
+        Accepts systemd specifiers (see systemd.unit(5)).
+      '';
+      default = [
+        "%h/.vscode-server/bin"
+        "%h/.vscode-server-oss/bin"
```

Comment:
I think we can remove this path again, I don't have any hope this will work (soon) with Codium.

Reply:
Removed the path but kept the option.